### PR TITLE
README is corrected

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

**Review by dkandrov — APPROVED** [review]

File: packages/NGSadd.taxonomy/meta/package.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <Package>
-	<DisplayName>Taxonomy classification (size Gb)</DisplayName>
+	<DisplayName>Taxonomy classification (3.1 Gb)</DisplayName>
```
Comment (dkandrov) [inline packages/NGSadd.taxonomy/meta/package.xml]: As I know size info shows automatically. And we doesn't need to add this info manually.
image

Comment (solomax) [reply packages/NGSadd.taxonomy/meta/package.xml]: Addressed in be511cd,
Thanks for pointing this out

**Comment by solomax** [comment]
Bad news :(((
Size in displayed in Windows installer :((

**Comment by dkandrov** [comment]
Maybe in new version of QT Installer Framework this was fixed. Unfortunately now I don't have access to a Linux machine to check new version of IFW.

**Comment by solomax** [comment]
I'm already using latest ....

**Pull request merged**